pages/api/cities/[city].js

```diff
@@ -63,10 +63,17 @@ handler.get(async (req, res) => {
 
   console.log(resp);
 
+  const response1 = await fetch(
```

Comment:
rather than calling this variable response1, maybe we could call it populationData?

In general it might be good to use more meaningful names throughout the code